config.toml

```diff
@@ -116,6 +116,12 @@ patterns = [ '^nfs' ]
   [nfs.scans.showmount]
   command = 'showmount -e {address} 2>&1 | tee "{result_file}"'
 
+[ntp]
+pattern = [ '^ntp' ]
+
+  [ntp.scans.nmap]
+  command = 'nmap -Pn -sV -p {port} --script="banner,ntp-info,ntp-monlist" -oN "{result_file}" {address}'
```

Comment:
I was thinking about that as well, but preferred to state the scripts explicitly. This prevents any script from running we don't actually know about. 
While the the config looks safe, I don't know, if it covers every possible combination of categories to prevent unintended scripts.